docs/book/src/topics/crd-changes/v1alpha7-to-v1alpha8.md

```diff
@@ -0,0 +1,43 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [v1alpha7 compared to v1alpha8](#v1alpha7-compared-to-v1alpha8)
+  - [Migration](#migration)
+  - [API Changes](#api-changes)
+    - [`OpenStackCluster`](#openstackcluster)
+      - [Change to externalNetworkID](#change-to-externalnetworkid)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# v1alpha7 compared to v1alpha8
+
+> ⚠️ v1alpha8 has not been released yet.
+
+## API Changes
+
+This only documents backwards incompatible changes. Fields that were added to v1alpha77777777 are not listed here.
+
+### `OpenStackCluster`
+
+#### Change to externalNetworkID
+
+The field `externalNetworkID` has been renamed to `externalNetwork` and isn't a string anymore but a reference to a `NetworkFilter` object.
+
+```yaml
+externalNetworkID: "..."
+```
+
+becomes
+
+```yaml
+externalNetwork:
+  id: "..."
+```
+
+It is now possible to specify a `NetworkFilter` object to select the external network to use for the cluster. The `NetworkFilter` object allows to select the network by name, by ID or by tags.
```

Comment:
Note for myself, the code doesn't allow to get the network by tags. Also I'll need to add a few validations to ensure we get one returned network.